includes/activity/class-activity.php

```diff
@@ -173,10 +173,13 @@ public function set_object( $object ) {
 	 */
 	public function get_json_ld_context() {
 		if ( $this->object instanceof Base_Object ) {
-			// Without php 5.6 support this could be just: 'return  $this->object::JSON_LD_CONTEXT;'
-			return call_user_func( array( get_class( $this->object ), 'CONTEXT' ) );
```

Comment:
Oh! Shouldn't it have been 'JSON_LD_CONTEXT' anyway? :/

Reply:
yes, but this has not solved it.